src/platform/forms/save-in-progress/FormStartControls.jsx

```diff
@@ -168,28 +167,31 @@ class FormStartControls extends React.Component {
 }
```

Comment:
Reverted the above from a `va-button` back to a `ProgressButton` since it is currently making numerous Cypress tests fail (selector is looking for a start "button") AND not completely removing it and only using a green action link as that would also make lots of changes to the Cypress test, and global changes to the form start element - which our team discussed doing A/B tests on.